examples/serving/serving.tensorflow.json

```diff
@@ -0,0 +1,21 @@
+{
+    "jobName": "tensorflow-serving",
+    "image": "paiexample/pai.example.tensorflow-serving",
+    "taskRoles": [
```

Comment:
For Service, set
maxRetryCount = -1 and fancyRetryPolicy = false
So, that it will always retry

Reply:
updated